geometry_msgs/Point[2] boundingBox
